monitor/utils/events.js

```diff
@@ -96,11 +97,40 @@ async function main(mode) {
       }
     })).map(normalizeEvent)
 
+    const foreignBridgeSwapped = new web3Foreign.eth.Contract([TOKENS_SWAPPED_EVENT_ABI], COMMON_FOREIGN_BRIDGE_ADDRESS)
+    const tokensSwappedEvents = (await getPastEvents(foreignBridgeSwapped, {
+      event: 'TokensSwapped',
+      fromBlock: MONITOR_FOREIGN_START_BLOCK,
+      toBlock: foreignBlockNumber
+    })).map(normalizeEvent)
+
+    // Get transfer events for each previous erc20
+    await Promise.all(
+      tokensSwappedEvents.map(async swap => {
+        const previousERC20 = new web3Foreign.eth.Contract(ERC20_ABI, swap.recipient)
```

Comment:
`recipient` is quite misleading. Does it make sense to implement another normalization method different from `normalizeEventInformation`?

Reply:
You're right, I removed the normalization, using the event directly is cleaner https://github.com/poanetwork/tokenbridge/pull/224/commits/a52b23cc0f4aa7e58bda9cf034d9059cf2192c67